src/Data/Repositories/Interfaces/IWalletRepository.cs

```diff
@@ -41,7 +41,7 @@ public interface IWalletRepository
 
     (bool, string?) RemoveRange(List<Wallet> types);
 
-    (bool, string?) Update(Wallet type, bool includeKeysUpdate = false);
+    Task<(bool, string?)> Update(Wallet type);
```

Comment:
Why make the update a task? Updates are synchronous always.

Reply:
I'm doing a async call to the database on file `src/Data/Repositories/WalletRepository.cs` line 142

Reply:
I mean, updates are sync in all the repos because EF Core Update operation is always sync, there's no need to overload with async if no async operation is really done